modules/ROOT/pages/demo/local-upload.adoc

```diff
@@ -3,16 +3,16 @@
 :keywords: example demo image local-upload
 :title_nav: Local file upload
 
-We've supported link:{baseurl}/demo/file-picker[local file picking] and link:{baseurl}/general-configuration-guide/upload-images[image uploading] for quite some time already. However making these features functional required some effort on the user's side, while not being immediately obvious, at the same time. Some effort is still required, but we decided to supplement the feature with an intuitive UI.
+We've supported xref:demo/file-picker.adoc[local file picking] and xref:general-configuration-guide/upload-images.adoc[image uploading] for quite some time already. However making these features functional required some effort on the user's side, while not being immediately obvious, at the same time. Some effort is still required, but we decided to supplement the feature with an intuitive UI.
 
 Once `images_upload_url` is defined in the config, an *Upload* tab appears in the *Image Dialog*.
 
-image::{baseurl}/images/demo_local-upload_01.png[Upload tab]
+image:demo_local-upload_01.png[Upload tab]
 
 So a setup like this displays a friendly UI simplifying image upload either by picking it up or by dropping it in, directly from the desktop:
 
 liveDemo::local-upload[height="400" ]
 
-All link:{baseurl}/general-configuration-guide/upload-images/#imageuploaderoptions[image uploading options] are supported, including `images_upload_handler` (a way to define custom file uploader) and `images_upload_credentials`.
+All xref:general-configuration-guide/upload-images.adoc#imageuploaderoptions[image uploading options] are supported, including `images_upload_handler` (a way to define custom file uploader) and `images_upload_credentials`.
```

Comment:
This xref doesn't resolve as the `imageuploaderoptions` id doesn't exist in `upload-images.adoc`. It looks like the id needs to be added to the relevant section title.

- [ ] Fix broken xref

Reply:
I added the needed id in the upload-images.adoc